target/classes/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
In general: never put generated (during the normal build process) files under version control.
This will only lead into merge conflicts and will not be helpful in any way

Reply:
So please remove the whole target folder from version control. (git rm .. and .gitignore)